packages/http/src/__tests__/fixtures/no-refs-petstore-minimal.oas2.json

```diff
@@ -16,28 +16,6 @@
               "default": "available"
             },
             "collectionFormat": "multi"
-          },
-          {
-            "in": "body",
```

Comment:
By not mocking the validation process anymore everywhere there was a test checking a completely different thing now starting to fail because of the body validation (which is now happening for real) — therefore I removed the body parameter at all so that the request goes through correctly as it should.